docs/application/tizen-studio/native-tools/manifest-text-editor.md

```diff
@@ -94,19 +109,20 @@ For more information on the relationship between the elements, see the [element
 
 **Expected children (in the following order):**
 
-| Child element           | Occurrences          |
-|-----------------------|--------------------|
-| `<author>`              | 1 (optional)         |
-| `<description>`         | 1 or more (optional) |
-| `<profile>`             | 1 or more (optional) |
-| `<ui-application>`      | 1 (optional)         |
-| `<service-application>` | 1 (optional)         |
-| `<watch-application>`   | 1 (optional)         |
-| `<widget-application>`  | 1 (optional)         |
-| `<shortcut-list>`       | 1 (optional)         |
-| `<account>`             | 1 (optional)         |
-| `<privileges>`          | 1 (optional)         |
-| `<feature>`             | 1 or more (optional) |
+| Child element                   | Occurrences          |
+|---------------------------------|----------------------|
+| `<author>`                      | 1 (optional)         |
+| `<description>`                 | 1 or more (optional) |
+| `<profile>`                     | 1 or more (optional) |
+| `<ui-application>`              | 1 (optional)         |
+| `<service-application>`         | 1 (optional)         |
+| `<watch-application>`           | 1 (optional)         |
+| `<widget-application>`          | 1 (optional)         |
+| `<component-based-application>` | 1 (optional)         |
+| `<shortcut-list>`               | 1 (optional)         |
+| `<account>`                     | 1 (optional)         |
+| `<privileges>`                  | 1 (optional)         |
+| `<feature>`                     | 1 or more (optional) |
 
```

Comment:
I've been fixed code indentation.
It doesn't effect about the link.
As I checked, all links don't work.
Please check it.

Reply:
@mijin85cho, request you to please provide your inputs on this.

Reply:
When we set up this repository, during the converting from html file to md file, we removed the links on purpose because of the readability and managing issue.
It is correct that now no links are there.

Reply:
@mijin85cho and @hjhun,

Thank you for your responses.